src/router/modules/Mapping.ts

```diff
@@ -0,0 +1,13 @@
+import Router from "@koa/router";
+import { MappingController } from "../../controllers/MappingController";
+
+export const mappingRouter = new Router();
+
+const mappingController = new MappingController();
+
+mappingRouter.get("/mapping/get-mapping", async (ctx, _) => {
+  const apiKey: string = ctx.get("Authorization").split(" ")[1] ?? "";
```

Comment:
implement the auth across all endpoints, not necessary now, can be done as new feature